src/converters/betterdiscord.ts

```diff
@@ -1,19 +1,37 @@
 import { Statement } from "@babel/types";
 import { IModImplementation } from "../api/ModImplementation.js";
-import { createFunctionFromObjectProperty } from "../api/RuntimeGenerators.js";
+import { createFunctionFromObjectProperty, createFunctionWithWrapperNeeded } from "../api/RuntimeGenerators.js";
 import { IBaseWebpackApi } from "../api/Webpack.js";
+import { IBasePatcherApi } from "../api/Patcher.js";
 
 class BDWebpackApi implements IBaseWebpackApi {
     get getModule() {
         return createFunctionFromObjectProperty("BdApi.Webpack", "getModule");
     }
 }
 
+class BDPatcherApi implements IBasePatcherApi {
+    get constructor_() {
+        return createFunctionWithWrapperNeeded("undefined", "undefined", "Patcher_constructor") as any;
+    }
+    internalId!: string;
+    get unpatchAll() {
+        return createFunctionWithWrapperNeeded("BdApi.Patcher", "unpatchAll", "unpatchAllWrapper");
+    }
+    get after() {
+        // return createFunctionWithWrapperNeeded("undefined", "undefined", "after");
```

Comment:
Remove debug code

Reply:
Solved in 75f11cde88db9e69536aec80def47096b37eefa6